### ISSUE TITLE
TypeError: Cannot read property 'require' of undefined

### ISSUE THREAD

**Issue by hackerxian** [JetBrains/svg-sprite-loader#366]
**Do you want to request a feature, report a bug or ask a question?**
report a bug
**What is the current behavior?**
```
Module build failed (from /xxx/node_modules/_svg-sprite-loader@4.1.6@svg-sprite-loader/lib/loader.js):
TypeError: Cannot read property 'require' of undefined
    at Object.<anonymous> (/xxxx/node_modules/_svg-sprite-loader@4.1.6@svg-sprite-loader/lib/utils/get-webpack-version.js:2:37)
```
**My webpack config**
```
{
        test: /\.svg$/,
        use: [
          {
            loader: require.resolve('svg-sprite-loader'),
            options: {
              symbolId: 'icon-[name]',
            },
          }
        ],
      }
```

```
import './icons/prompt.svg';
```

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please tell us about your environment:**

- Node.js version: v8.12.0
- webpack version: 4.41.2
- svg-sprite-loader version: 4.1.6
- OS type & version: Mac 10.14.6 

**Other information**
`get-webpack-version.js` use  `require` replace `require.main`, it works.

**Comment by xidb** [comment]
Made a PR for that here https://github.com/JetBrains/svg-sprite-loader/pull/382

**Comment by kisenka** [comment]
Should be fixed in `svg-sprite-loader@4.2.6`
